src/test/java/org/folio/rest/impl/PatronResourceImplTest.java

```diff
@@ -1020,6 +1031,61 @@ public final void testPostPatronAccountByIdInstanceByInstanceIdHoldWithErrors(
     logger.info("Test done");
   }
 
+  @Test
+  public final void testPostPatronAccountByIdInstanceByInstanceIdHold422Error() {
+    logger.info("Testing creating a hold on an instance for the specified user with bad instance id error");
+
+    final var holdErrorResponse = given()
+        .headers(new Headers(tenantHeader, urlHeader, contentTypeHeader,
+            new Header("x-okapi-bad-instance-id", badInstanceId),
+            new Header(okapiBadDataHeader, "422")))
+        .and().pathParams("accountId", goodUserId, "instanceId", badInstanceId)
+        .and().body(readMockFile(mockDataFolder
+        + "/request_testPostPatronAccountByIdInstanceByInstanceIdHold.json"))
+      .when()
+        .contentType(ContentType.JSON)
+        .post(accountPath + instancePath + holdPath)
+        .then()
+        .log().all()
+        .and().assertThat().contentType(ContentType.JSON)
+        .and().assertThat().statusCode(422)
+      .extract()
+        .as(Errors.class);
+
+    final var expectedErrors = Json.decodeValue(readMockFile(mockDataFolder + "/instance_hold_bad_instance_id.json"), Errors.class);
+    assertEquals(expectedErrors, holdErrorResponse);
+
+    // Test done
+    logger.info("Test done");
```

Comment:
If the test doesn't finish, the build would stall. What extra information does this log message provide?